packages/app-desktop/gui/NoteEditor/NoteEditor.tsx

```diff
@@ -190,7 +202,7 @@ function NoteEditor(props: NoteEditorProps) {
 
 		setScrollWhenReady({
 			type: props.selectedNoteHash ? ScrollOptionTypes.Hash : ScrollOptionTypes.Percent,
-			value: props.selectedNoteHash ? props.selectedNoteHash : props.lastEditorScrollPercents[props.noteId] || 0,
+			value: props.selectedNoteHash ? props.selectedNoteHash : props.lastEditorScrollPercents[formNote.id] || 0,
```

Comment:
Why was this line change? Isn't it a separate bug and, if so, could it be a separate pull request? I see that whole effect is pretty bad anyway, but if we can tweak it in a separate pull request it's for the best.

Reply:
It's a part of the fix of the same bug.
As the other two `props.noteId` are replaced into `effectiveNoteId`, this `props.noteId` is replaced into `formNote.id`. Since it is in a useEffect closure, `formNote.id` should be used instead of `effectiveNoteId`.